ansible/roles/postgresql_server/templates/usr/sbin/autopostgresqlbackup.j2

```diff
@@ -290,11 +290,19 @@ encryption() {
     echo "Encrypting $1"
     echo "      to $ENCRYPTED_FILE"
     echo "      using cypher $ENCRYPTION_CIPHER and public key $ENCRYPTION_PUBLIC_KEY"
-    if openssl smime -encrypt -"$ENCRYPTION_CIPHER" -binary -outform DEM \
-      -out "$ENCRYPTED_FILE" \
-      -in "$1" "$ENCRYPTION_PUBLIC_KEY" ; then
+
+    tempcrt=/tmp/autopostgresql.crt
```

Comment:
We can have more than one `autopostgresqlbackup` scripts running. Perhaps it would be better to go with
```bash
tempcrt="$(tempfile)"
```
here so that we know that the filename is unique and not taken by anything else?

Reply:
```
In ./ansible/roles/postgresql_server/templates/usr/sbin/autopostgresqlbackup.j2 line 294:
    tempcrt=$(tempfile)
              ^------^ SC2186: tempfile is deprecated. Use mktemp instead.
```

me and shellcheck have not been getting on :rofl: